frontend/src/icons/close-query-tab.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
let's run [svgo](https://github.com/svg/svgo) on the icons. You convert them to components, but I think will still help to make them more optimal.

Reply:
Before modifying these icons, let's confirm with @ricardobaeta's it's safe.

Are these files the sources you would use to edit, or just exported versions? in other words, if we modify these files, would that prevent you from going into this repository to tweak some icons?

Reply:
Hi Carlos, as I understood the icons are Free, you just can't put it in your website - the originals -  as a resource without noticing the designer. We used the icons and we've changed every one of them. They will be in this repo yes, but we're not redistributing the designer's work. Do you agree? @marnovo What's your take on this?

Reply:
In this thread what I'm asking is not really related to the license. What Maxim is suggesting is to modify the `.svg` files here to make them simpler and smaller (internal structure only, without changing the resulting image). And what I'm asking you to confirm is whether or not you would care if we do something to these files that prevents you from opening them, and working in this repository files directly.

I assume you have your files somewhere else and these are just exports that you don't plan to open and modify directly, but just to double check. Thanks!